File: Entity+CoreDataProperties.m
//
//  Entity+CoreDataProperties.m
//  
//
//  Created by topsec on 17/1/10.
//
//

#import "Entity+CoreDataProperties.h"

@implementation Entity (CoreDataProperties)

+ (NSFetchRequest<Entity *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Entity"];
}

@dynamic uuid;
@dynamic tasktage;
@dynamic imag;

@end

File: Entity+CoreDataProperties.swift
//
//  Entity+CoreDataProperties.swift
//  
//
//  Created by topsec on 17/1/10.
//
//

import Foundation
import CoreData


extension Entity {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Entity> {
        return NSFetchRequest<Entity>(entityName: "Entity");
    }

    @NSManaged public var uuid: String?
    @NSManaged public var tasktage: String?

}
